### PR TITLE
API definition name passed to ClusterAssetGroup

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**

Changes proposed in this pull request:

- Compass Runtime Agent sets API name in ClusterAssetGroup

**Related issue(s)**
https://github.com/kyma-project/console/issues/1617

### CODE REVIEW

File: components/compass-runtime-agent/internal/kyma/util.go [hunk 1]
```diff
@@ -10,22 +10,24 @@ import (
...
 
 	return clusterassetgroup.Asset{
-		ID:      apiDefinition.ID,
-		Name:    apiDefinition.Name,
-		Type:    getApiType(apiDefinition.APISpec),
-		Content: getSpec(apiDefinition.APISpec),
-		Format:  getSpecFormat(apiDefinition.APISpec),
+		ID:          apiDefinition.ID,
+		Name:        apiDefinition.Name,
+		DisplayName: apiDefinition.Description,
```
Comment (akucharska) [inline components/compass-runtime-agent/internal/kyma/util.go:27]: Can you use name from APIDefinition/EventAPIDefinition? 
Screen Shot 2020-04-07 at 10 23 13
Screen Shot 2020-04-07 at 10 22 00

File: components/compass-runtime-agent/internal/kyma/util.go [hunk 2]
```diff
@@ -10,22 +10,24 @@ import (
...
 
 	return clusterassetgroup.Asset{
-		ID:      eventAPIDefinition.ID,
-		Name:    eventAPIDefinition.Name,
-		Type:    getEventApiType(eventAPIDefinition.EventAPISpec),
-		Content: getEventSpec(eventAPIDefinition.EventAPISpec),
-		Format:  clusterassetgroup.SpecFormat(getEventSpecFormat(eventAPIDefinition.EventAPISpec)),
+		ID:          eventAPIDefinition.ID,
+		Name:        eventAPIDefinition.Name,
+		DisplayName: eventAPIDefinition.Description,
```
Comment (rafalpotempa) [inline components/compass-runtime-agent/internal/kyma/util.go:15]:
```suggestion
		DisplayName: eventAPIDefinition.Name,
```

File: tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go [hunk 3]
```diff
@@ -210,6 +210,12 @@ func (c *K8sResourceChecker) assertAssetGroup(t *testing.T, apiPackage *graphql.
 			source, found := getAPISource(api.ID, assetGroup)
 			require.True(t, found)
 
+			apiDescription := ""
+			if api.Description != nil {
+				apiDescription = *api.Description
+			}
+			assert.Equal(t, apiDescription, source.DisplayName)
```
Comment (rafalpotempa) [inline tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go:217]: I'm not sure if above comments yield additional changes here...

File: tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go [hunk 4]
```diff
@@ -219,6 +225,12 @@ func (c *K8sResourceChecker) assertAssetGroup(t *testing.T, apiPackage *graphql.
 			source, found := getAPISource(eventAPI.ID, assetGroup)
 			require.True(t, found)
 
+			eventApiDescription := ""
+			if eventAPI.Description != nil {
+				eventApiDescription = *eventAPI.Description
+			}
+			assert.Equal(t, eventApiDescription, source.DisplayName)
```
Comment (rafalpotempa) [inline tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go:232]: I'm not sure if above comments yield additional changes here...

File: components/compass-runtime-agent/internal/kyma/util.go (hunk 1 above)
Comment (akgalwas) [reply components/compass-runtime-agent/internal/kyma/util.go]: Done.

File: tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go (hunk 3 above)
Comment (akgalwas) [reply tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go]: Done

File: tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go (hunk 4 above)
Comment (akgalwas) [reply tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go]: Done

File: components/compass-runtime-agent/internal/kyma/util.go (hunk 2 above)
Comment (akgalwas) [reply components/compass-runtime-agent/internal/kyma/util.go]: Done

**Comment by franpog859** [comment]
/test pre-master-kyma-gke-integration
here is the [previous log](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/7913/pre-master-kyma-gke-integration/1248114176920915968)

**Pull request merged**